pkg/version/version.go

```diff
@@ -38,8 +38,8 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                  = "0.11.0"
-	PrevReleaseVersion         = "0.10.0"
```

Comment:
@cevian should this be `0.10.` or `0.11.0` for a patch release?

Reply:
(I'm not Mat, but I think it should be 0.11.0)

Reply:
0.11.0

Reply:
Does that mean upgrade from 0.10.0 to 0.11.1 is not possible?